hackathon_site/hardware/migrations/0001_initial.py

```diff
@@ -0,0 +1,116 @@
+            ],
+        ),
+        migrations.CreateModel(
+            name="Hardware",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                ("model_number", models.CharField(max_length=255)),
+                ("manufacturer", models.CharField(max_length=255)),
+                ("datasheet", models.URLField()),
+                ("quantity_available", models.IntegerField()),
+                ("notes", models.TextField()),
+                ("max_items_per_team", models.IntegerField()),
+                ("picture", models.FileField(upload_to="hardware/pictures/")),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                ("categories", models.ManyToManyField(to="hardware.Category")),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Order",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("part_returned_health", models.CharField(max_length=64, null=True)),
+                ("status", models.CharField(max_length=64)),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                (
+                    "hardware",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="hardware.Hardware",
+                    ),
+                ),
+                (
+                    "team",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="event.Team"
+                    ),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Incident",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("state", models.CharField(max_length=64)),
+                ("time_occurred", models.DateTimeField()),
+                ("location_occurred", models.CharField(max_length=255)),
```

Comment:
I didn't include location on the mockup form. If you want to include it, I think it should be a text field. Tbh I don't think it's important to keep track of.